client/spec.yaml

```diff
@@ -495,4 +495,178 @@ paths:
+        "5XX":
+          description: Server side error
+  /api/logs/v1/{tenant}/loki/api/v1/rules/{namespace}/{group}:
+    parameters:
+      - $ref: ./parameters/parameters.yaml#/components/parameters/tenant
+      - $ref: ./parameters/parameters.yaml#/components/parameters/logRulesNamespace
+      - $ref: ./parameters/parameters.yaml#/components/parameters/logRulesGroup
+    get:
+      tags:
+        - logs/rulesv1
+      summary: Get Logs Rules for a tenant in JSON form inside a namespace and a group
+      operationId: getLogsRulesGroup
+      description: |
+        You can get Logs rules for a tenant in a namespace and a group with health data
+      responses:
+        "2XX":
+          description: rule for a tenant
+          content:
+            application/json:
+              schema:
+                $ref: "./responses/responses.yaml#/components/schemas/LogRulesGroupResponse"
+        "4XX":
+          description: Bad request
+        "5XX":
+          description: Server side error
+    delete:
+      tags:
+        - logs/rulesv1
+      summary: Delete Logs Rules for a tenant in JSON form inside a namespace and a group
+      operationId: deleteLogsRulesGroup
+      description: |
+        You can delete Logs rules for a tenant in a namespace and a group
+      responses:
+        "2XX":
+          description: rule for a tenant
+        "4XX":
+          description: Bad request
+        "5XX":
+          description: Server side error
+  /api/logs/v1/{tenant}/prometheus/api/v1/rules:
+    parameters:
+      - $ref: ./parameters/parameters.yaml#/components/parameters/tenant
+    get:
+      tags:
+        - logs/rulesv1
+      summary: Prometheus-compatible rules endpoint to list alerting and recording rules that are currently loaded.
+      operationId: getLogsPromRules
+      description: |
+        Prometheus-compatible rules endpoint to list alerting and recording rules that are currently loaded.
+      responses:
+        "2XX":
+          description: rule for a tenant
+          content:
+            application/json:
+              schema:
+                $ref: "./responses/responses.yaml#/components/schemas/LogRulesPrometheusResponse"
+        "4XX":
+          description: Bad request
+        "5XX":
+          description: Server side error
+  /api/logs/v1/{tenant}/prometheus/api/v1/alerts:
+    parameters:
+      - $ref: ./parameters/parameters.yaml#/components/parameters/tenant
+    get:
+      tags:
+        - logs/rulesv1
+      summary: Prometheus-compatible rules endpoint to list all active alerts.
+      operationId: getLogsPromAlerts
+      description: |
+        Prometheus-compatible rules endpoint to list all active alerts.
+      responses:
+        "2XX":
+          description: rule for a tenant
+          content:
+            application/json:
+              schema:
+                $ref: "./responses/responses.yaml#/components/schemas/LogRulesPrometheusResponse"
```

Comment:
would we need to refer here to the alert rules schema instead? https://github.com/observatorium/api/blob/16e21d560de0f56477a533d4701427e3ede6584a/client/models/models.yaml#L117

Reply:
The chain is `LogRulesPrometheusResponse` -> `Rules` -> `RuleGroupEvaluated` and the last one has:
```
        rules:
          type: array
          items:
            oneOf:
              - $ref: "#/components/schemas/RecordingRuleEvaluated"
              - $ref: "#/components/schemas/AlertingRuleEvaluated"
```